src/ape/managers/project/types.py

```diff
@@ -265,7 +265,7 @@ def _compile_sources(proj_srcs: _ProjectSources) -> dict[str, ContractType]:
                 self._cached_manifest.contract_types = filtered_contract_types
                 self._contracts = filtered_contract_types
 
-            return self.compiler_manager.compile(srcs_to_compile)
+            return {ct.name: ct for ct in self.compiler_manager.compile(srcs_to_compile) if ct.name}
```

Comment:
If there isn't a name defined, it will skip?

Also, would it be better to do `ct.name is not None` (I think it can be `None`)

Reply:
81ef891